### PR TITLE
Fixes #223 - Executive Report just spins if there is nothing to show

### LINKED ISSUE #233: Executive Report just spins if there is nothing to show
While demoing to DSG, a new evaluation was created for them and they added 60+ systems. When using the web app, they selected Executive Reports and were presenting with a loading spinner that never went away. I believe this is because the executive reports did not have enough time to complete but we should show them a message to indicate this.

### PR DESCRIPTION
**Pull request by Kinlaw**
Fixes #233. This also fixes spacing in the systems under 10 message.

@jharting this is basically just one if statement. The reason for so many lines being changed is I wrapped the all of the initial code that is ran in a getData function so that I could break out of it if there was no data.

### CODE REVIEW

File: app/js/states/digests/digests.jade [hunk 1]
```diff
@@ -20,10 +20,15 @@
...
+        h2(translate) Please register&nbsp;
           span(translate, translate-n='systemsTilTen', translate-plural='{{$count}} more systems') {{systemsTilTen}} more system
           | &nbsp;to enable reporting for your account.
-  .content-wrap(ng-show='!loading && systemsTilTen < 1')
+  .content-wrap.margin-top(ng-show='!loading && noData')
+    .row
+      .col-sm-12.col-md-8.col-md-offset-2.text-center
+        h1(translate) Executive Reporting does not currently have any data for this account
+        h2(translate) Please check back later.
+  .content-wrap(ng-if='!loading && systemsTilTen < 1 && !noData')
```
Comment (Kinlaw) [inline app/js/states/digests/digests.jade]: @mattnolting @Robcwilliams what message should we show the user if there is no data to show? This happens whenever a new account is created and digests haven't run yet.

**Comment by Kinlaw** [comment]
Screenshot of error message
screen shot 2017-11-28 at 10 09 31 am

**Comment by iphands** [comment]
@mattnolting this seems like a k.i.s.s. fix... any objections from what @Kinlaw is doing presentation wise?

File: app/js/states/digests/digests.jade (hunk 1 above)
Comment (Robcwilliams) [reply app/js/states/digests/digests.jade]: > Your initial executive report is currently processing - this may take some time, please check back later.

**Pull request merged**